docs/observability/aws/quickstart.md

```diff
@@ -5,21 +5,48 @@ sidebar_label: AWS Observability Quickstart
 description: Get up and running in minutes with our AWS Observability integration.
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
 This guide will walk you through setting up [Sumo Logic's AWS Observability integration](https://www.sumologic.com/application/aws-observability/)
 
-:::note Prerequisites
-You will need the [AWS CLI](https://aws.amazon.com/cli/) command installed and configured for your AWS account
-:::
+## Prerequisites
+You will need the [AWS CLI](https://aws.amazon.com/cli/) command installed and configured for your AWS account.
+
 
 ## Installation
 
 Get your [Sumo Logic Access ID and Access Key](/docs/manage/security/access-keys) and run the following command:
 
+<Tabs
+className="unique-tabs"
+defaultValue="Windows"
+values={[
+{label: 'Windows', value: 'Windows'},
+{label: 'Linux / Mac', value: 'Linux / Mac'}
+]}>
+
+<TabItem value="Windows">
+
+Copy the file available here. Save it as “DeployAWSOWin.ps1”. Run the command:
+
+```
+.\DeployAWSOWin.ps1 <SUMO_ACCESS_ID> <SUMO_ACCESS_KEY>
+```
+
+
+</TabItem>
+<TabItem value="Linux / Mac">
+
+For :
+Copy the file available here. Save it as “DeployAWSOPosix.sh”. Run the command:
```

Comment:
here - should have an embedded link which I had pasted in my suggested doc. Its missed here. Here is the link - https://github.com/SumoLogic/sumologic-solution-templates/blob/master/aws-observability/scripts/AWSOAutoSetupScript/DeployAWSOPosix.sh